create ProductDetail.jsx and add to App,jsx

jdougross opened the pull request:
added a Product Detail component file with a hello world.  Will develop this file folder as we go.

Requested review from any of you, hopefully this won't require EVERYONE to review it.... apologies if it does.

jdougross committed:
create ProductDetail.jsx and add to App,jsx

Kevinfante reviewed (changes requested):
Everything looks good so far. Might want to go ahead and start looking into styling using css modules. That would require some changes made to the css sheet, importing the styles sheet into your file and using jsx to  style your module accordingly. Other than that looks good

jdougross committed:
productDetail section linted

jdougross commented:
Perfect.  Addressed that without tweaking style.css by putting a placeholder object at the top.  That way when it comes time for css, we just swap that object out for the stuff we import.  Also, I fixed any linter errors I found.

Kevinfante reviewed (approved):
Okay looks good now. I've gone ahead and approved.
